Docs: Contributing folder (#4415)
langflow-ai/langflow: docs/docs/Contributing/contributing-github-issues.md

---
title: Request an enhancement or report a bug
sidebar_position: 2
slug: /contributing-github-issues
---

> The [Issues page in the Langflow repo](https://github.com/langflow-ai/langflow/issues) is kept up to date with bugs, improvements, and feature requests. Labels are used to help with sorting and discovery of issues of interest. For an overview of the system Langflow use to tag issues and pull requests, see the Langflow repo's [labels page](https://github.com/langflow-ai/langflow/labels).

Reviewer: ```suggestion
The [Issues page in the Langflow repo](https://github.com/langflow-ai/langflow/issues) is kept up to date with bugs, improvements, and feature requests. Labels are used to help with sorting and discovery of issues of interest. For an overview of the system Langflow uses to tag issues and pull requests, see the Langflow repo's [labels page](https://github.com/langflow-ai/langflow/labels).
```

slug: /contributing-github-issues
---

> Our [issues](https://github.com/langflow-ai/langflow/issues) page is kept up to date with bugs, improvements, and feature requests. There is a taxonomy of labels to help with sorting and discovery of issues of interest. For an overview of the system we use to tag our issues and pull requests, see the Langflow repo's [labels page](https://github.com/langflow-ai/langflow/labels).

Reviewer: ```suggestion
The [Issues page in the Langflow repo](https://github.com/langflow-ai/langflow/issues) is kept up to date with bugs, improvements, and feature requests. Labels are used to help with sorting and discovery of issues of interest. For an overview of the system Langflow use to tag issues and pull requests, see the Langflow repo's [labels page](https://github.com/langflow-ai/langflow/labels).
```

> ## GitHub Discussions

Reviewer: ```suggestion
## Ask a question on the Discussions page
```

> ## GitHub Discussions

Reviewer: I'm not sure why this is included in information about contributing. I this this would be better in an FAQ under "How can I get help?

Author: Moved to a new page in Contributing

> If you're looking for help with your code, consider posting a question on the Langflow [GitHub Discussions board](https://github.com/langflow-ai/langflow/discussions). Please understand that we won't be able to provide individual support via email. We also believe that help is much more valuable if it's **shared publicly**, so that more people can benefit from it.

Reviewer: ```suggestion
If you're looking for help with your code, consider posting a question on the Langflow [GitHub Discussions board](https://github.com/langflow-ai/langflow/discussions). The Langflow team cannot provide individual support via email. We also believe that help is much more valuable if it's **shared publicly**, so that more people can benefit from it.
```

Since the Discussions board is public, please follow this guidance when posting your code questions.

1. When describing your issue, try to provide as many details as possible. What exactly goes wrong? _How_ is it failing? Is there an error? "XY doesn't work" usually isn't that helpful for tracking down problems. Always remember to include the code you ran and if possible, extract only the relevant parts and don't just dump your entire script. This will make it easier for us to reproduce the error.

> 2. When you include long code, logs, or tracebacks, wrap them in `<details>` and `</details>` tags. This [collapses the content](https://developer.mozilla.org/en/docs/Web/HTML/Element/details) so the contents only becomes visible on click, making the issue easier to read and follow.

Reviewer: These items should be in a bulleted list, not a numbered list. The order is not important here.
